src/core_plugins/kibana/public/visualize/listing/visualize_listing.html

```diff
@@ -32,26 +32,11 @@
 
       <div class="kuiToolBarSection">
         <!-- Bulk delete button -->
-        <button
-          class="kuiButton kuiButton--danger"
-          aria-label="Delete selected objects"
-          ng-if="listingController.getSelectedItemsCount() > 0"
-          ng-click="listingController.deleteSelectedItems()"
-          tooltip="Delete selected visualizations"
-        >
-          <span aria-hidden="true" class="kuiButton__icon kuiIcon fa-trash"></span>
-        </button>
+        <react-component name="TrashButtonComponent" props="trashButtonProps"></react-component>
```

Comment:
I ended up getting rid of the need for ng-if by combining these both into one react component.  The main reason being the tooltip prop conflicted with the angular one and both tooltips were showing.